index.html

```diff
@@ -108,7 +108,7 @@ <h3>Let's see what the Altventurers have made for you</h3>
             />
             Download Full Project PDF</a
           >
-          <a id="footer-download-a11y" class="download-button" href="#">
+          <a id="footer-download-a11y" class="download-button" href="https://docs.google.com/document/d/1JNKdL4Js6JpJqumtm6uyyYqW_xKRv2OhT6fgoJWwK3g">
```

Comment:
assuming  this is still pending the pdf download link

could you also add target="_blank" to each of the links so they open in a new tab?

Reply:
Gotcha, thanks!